api/lib/infrastructure/repositories/higher-education-registration-repository.js

```diff
@@ -0,0 +1,55 @@
+const { knex } = require('../bookshelf');
+const _ = require('lodash');
+
+const { SchoolingRegistrationsCouldNotBeSavedError } = require('../../domain/errors');
+
+module.exports = {
+
+  async saveSet(higherEducationRegistrationSet) {
+
+    const studentNumbers = [];
+    const names = [];
+    higherEducationRegistrationSet
+      .registrations()
+      .forEach(({ studentNumber, firstName, lastName, birthdate }) => {
+        studentNumbers.push(studentNumber);
+        names.push([firstName, lastName, birthdate]);
+      });
+
+    const countOfAlreadyExistingSchoolingRegistration = await knex('schooling-registrations')
+      .whereIn('studentNumber', studentNumbers)
+      .andWhere({ organizationId: higherEducationRegistrationSet.organizationId })
+      .count();
+
+    if (countOfAlreadyExistingSchoolingRegistration[0].count > 0) {
+      throw new SchoolingRegistrationsCouldNotBeSavedError();
+    }
+
+    const schoolingRegistrationRows = higherEducationRegistrationSet
+      .registrations()
+      .map((registration) => {
+        const attributs =  _.pick(registration, [
+          'organizationId',
+          'studentNumber',
+          'firstName',
+          'middleName',
+          'thirdName',
+          'lastName',
+          'preferredLastName',
+          'studentNumber',
+          'email',
+          'birthdate',
+          'group'
+        ]);
+        return {
+          ...attributs,
+          educationalTeam: registration.teachingTeam,
```

Comment:
```suggestion
          educationalTeam: registration.educationalTeam,
```
je trouve ça plus pertinent de garder le même terme

Reply:
done